CollAction/Services/ProjectService.cs

```diff
@@ -84,6 +85,7 @@ public async Task<IEnumerable<FindProjectsViewModel>> FindProjects(Expression<Fu
                     {
                         ProjectId = project.Id,
                         ProjectName = project.Name,
+                        ProjectUriPart = project.Name.NormalizeUriPart(),
```

Comment:
Maybe call it ProjectNameUriPart? Or ProjectNameUri? Seems a bit more descriptive.